WordPress/openverse
# Remove client application headers from API responses

## Fixes



Fixes #4569 by @sarayourfriend 

## Description


The `response_headers_middleware` is removed given we don't want to include these headers anymore.

## Testing Instructions

1. Run  `just api/up` and a register a new application in the API
2. watch the logs

```
just logs | grep client_application_authentication
```

3. and make an authed request to observe the line.

```
2024-07-19 17:29:16 2024-07-19T21:29:16.170240Z [info     ] client_application_authentication [conf.oauth2_extensions] application_id=1 application_name=Testing App application_verified=False filename=oauth2_extensions.py func_name=authenticate ip=192.168.65.1 lineno=37 request_id=84d16929-4328-4cc0-94ff-6e1f1f365fe1
```

## Checklist



[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
